### ISSUE TITLE
[FEATURE REQUEST] WebSearch plugin >> Live Search Preview

### ISSUE THREAD

**Issue by sstar0016** [Flow-Launcher/Flow.Launcher#1992]
## Is your feature request related to a problem? Please describe.
Not a problem, but would love if we could get the same functionality that native windows search gives us, showing a web quick response that means I don't have to actually swap over to the browser. 

## Describe the solution you'd like
Would just replace the preview pane that currently only shows the search engine icon.
![image](https://user-images.githubusercontent.com/124852454/226168803-3e15f172-9f3e-4899-97a4-928193645383.png)
![image](https://user-images.githubusercontent.com/124852454/226168776-3a256809-3133-4008-b874-61729421000e.png)
- bonus would be if we could scroll down the preview page and click links/ interact (as you can do with the native search)
![image](https://user-images.githubusercontent.com/124852454/226169131-55cd1cce-1f67-491a-b6ca-19e53f88ec42.png)
![image](https://user-images.githubusercontent.com/124852454/226169278-0945bd2d-bd67-4abc-8cc9-401605acf4dd.png)

## Other Solutions
I know I could use the unit converter plugin and maybe people will develop news plugins etc in the future but it would be simpler for me if this preview option does it all

## Additional context
I use this regularly to check stuff live (obscure conversions, stocks, scores, news) but am trying to swap from the native taskbar search to Flow-Launcher, so would really appreciate if this could be implemented.

**Comment by nikolicpetar17** [comment]
Great idea in my opinion

**Comment by nikolicpetar17** [comment]
(just to remove stale label)

**Comment by onesounds** [comment]
We tried to implement this feature, but the search engines usually blocked showing webpages like that. (Fluent search implemented this feature, but for the same reason, they show mobile site results. I decided it wasn't useful to show mobile sites that way. For example, BING results in the Start menu, we provide search result output rendered to that size on the BING site. They can do this because it's their service. we tried to use Google, and we can't get the Google search results like that.) Each site needs its own server or parsing logic to display beautiful results. This is out of our scope, so we have no plans to implement it at this time.
